README.md

```diff
@@ -53,6 +53,15 @@ replacing `branch` as appropriate
 gclient config --name="certificate-transparency" https://github.com/google/certificate-transparency.git@branch
 ```
 
+### Language Specific Notes ###
+
+#### Java ####
```

Comment:
Yeah, and in the "old" Ubuntu quickstart, it had you install `libprotobuf-java`, which depends on `protobuf-compiler` (which was also listed explicitly anyway).

This is the "old" stuff, because the "new" stuff is gclient, but Java isn't included in gclient (and if you have me build a JVM and Java libraries as part of building a CT server, I might have to lodge a formal complaint of some sort). :wink: